frontend/dialogs/OBSIdianPlayground.cpp

```diff
@@ -0,0 +1,142 @@
+/******************************************************************************
+    Copyright (C) 2023 by Dennis Sädtler <dennis@obsproject.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#include <QTimer>
+
+#include <Idian/Idian.hpp>
+#include "OBSIdianPlayground.hpp"
```

Comment:
Apply the include order introduced with the frontend framework:

```C++
#include "OBSIdianPlayground.hpp" // Counterpart
#include "SomeOtherFileInTheSameDirectory.hpp"
#include <Some/OtherFileInSameTargetButDifferentDirectory.hpp>

#include <Idian/Idian.hpp>
#include <Some/OtherFileFromAnotherTarget.hpp>

#include <3rdPartyLibraryHeader.hpp>

#include <StandardLibraryHeader.hpp>
```

This should be applied to all files (I've seen more instances of the order not being followed below).